### PR TITLE
5808 yhdistelmaopintojaksot

### LINKED ISSUE #5808: Yhdistelmäopintojaksot

### PR DESCRIPTION
**Pull request by aviljakaine**
Closes #5808

### CODE REVIEW

**Review by onzag — DISMISSED** [review]

File: updates_devel/220321_supplementationrequest_workspacesubject.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<update xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.ofw.fi/xml/2011/java-xmldb-updater/UpdaterSchema.xsd">
```
Comment (pkukkon) [inline updates_devel/220321_supplementationrequest_workspacesubject.xml:2]: Remember to change to official database version.

Comment (pkukkon) [inline updates_devel/220321_supplementationrequest_workspacesubject.xml:2]: Remember to change to official database version.

**Review by pkukkon — APPROVED** [review]
Backend OK (with updated db script)

**Review by avukkonen — CHANGES_REQUESTED** [review]

**Review by pkukkon — APPROVED** [review]
Backend OK

**Review by avukkonen — DISMISSED** [review]

**Review by pkukkon — APPROVED** [review]

**Pull request merged**